test/utilities/navigationUtilities.spec.js

```diff
@@ -10,3 +17,26 @@ ava('assembleProceduralRouteString() concatenates provided strings with appropri
 
     t.true(result === expectedResult);
 });
+
+ava('getCircularCoordinates() provides a set of points equidistant from center', (t) => {
+    const TOLERABLE_ERROR = 0.025  // 25m
+
+    const posAndDecl = [airportModelFixture.positionModel, MAGNETIC_NORTH_MOCK];
+    const center = GPS_COORDINATES_MOCK;
```

Comment:
You shouldn't reassign and rename constant like this.  you're loosing context and opening the door to modification.  just use `GPS_COORDINATES_MOCK`